indexer/editable_map_object.hpp

```diff
@@ -56,7 +56,19 @@ struct NamesDataSource
   vector<LocalizedName> names;
   size_t mandatoryNamesCount = 0;
 };
-  
+
+struct FakeNameInfo
+{
+  int8_t code;
+  string filledName;
+};
+// Contains information about fake names which were added for user convenience.
+struct FakeNames
+{
+  vector<FakeNameInfo> names;
+  string defaultName;
```

Comment:
Why do you need a separate field for default name? Why not to just always have at least one element in names, with default name as filledName and default language code as code?